examples/performance_benchmarks.py

```diff
@@ -229,12 +283,31 @@ def run_linear_qubit_operator(n_qubits=16, n_terms=10, processes=10):
     _, runtime_parallel = benchmark_linear_qubit_operator(n_qubits, n_terms,
                                                           processes)
     logging.info('LinearQubitOperator took %f seconds, while '
-                 'ParallelQubitOperator took %f seconds with %d processes, and '
-                 'ratio is %.2f.\n', runtime_sequential, runtime_parallel,
+                 'ParallelQubitOperator took %f seconds with %d processes, '
+                 'and ratio is %.2f.\n', runtime_sequential, runtime_parallel,
                  processes, runtime_sequential / runtime_parallel)
 
     return runtime_sequential, runtime_parallel
 
+
+def run_diagonal_commutator(side_length=4):
+    """Run commutator_diagonal_coulomb_operators benchmark."""
```

Comment:
Can you explain a bit more what this benchmark is about?

Reply:
This just gives a small demo to show the speed difference between `commutator` and the new function. @kevinsung suggested adding one, I think it's nice to see / makes the speedup a bit more tangible. The benchmark splits the terms from a jellium Hamiltonian in two, and computes the commutator of the two parts.